Tests/SonarScanner.MSBuild.PreProcessor.Test/TargetsInstallerTests.cs

```diff
@@ -281,8 +278,10 @@ private void InstallLoaderTargets_InternalCopyTargetFileToProject(string sourceC
         {
             // Arrange
             var targetsInstaller = new TargetsInstaller(this.logger, this.msBuildPathSettingsMock.Object, this.fileWrapperMock.Object, this.directoryWrapperMock.Object);
-            var sourcePath = Path.Combine(TestContext.DeploymentDirectory, "Targets", "SonarQube.Integration.targets");
-
+            var sourcePath = Path.Combine(
```

Comment:
the failure for the 3 left ignored tests happens at 

- line 305 
```
Expected invocation on the mock once, but was 0 times: x => x.Copy("C:\workspace\sonar-scanner-msbuild\tests\SonarScanner.MSBuild.PreProcessor.Test\bin\Debug\net48\Targets\SonarQube.Integration.ImportBefore.targets", "c:\project\bin\targets\SonarQube.Integration.targets", True)

Performed invocations:

   Mock<IFileWrapper:1> (x):

      IFileWrapper.Exists("C:\workspace\sonar-scanner-msbuild\tests\SonarScanner.MSBuild.PreProcessor.Test\bin\Debug\net48\Targets\SonarQube.Integration.ImportBefore.targets")
      IFileWrapper.ReadAllText("C:\workspace\sonar-scanner-msbuild\tests\SonarScanner.MSBuild.PreProcessor.Test\bin\Debug\net48\Targets\SonarQube.Integration.ImportBefore.targets")
      IFileWrapper.Exists("C:\workspace\sonar-scanner-msbuild\tests\SonarScanner.MSBuild.PreProcessor.Test\bin\Debug\net48\Targets\SonarQube.Integration.targets")
      IFileWrapper.ReadAllText("C:\workspace\sonar-scanner-msbuild\tests\SonarScanner.MSBuild.PreProcessor.Test\bin\Debug\net48\Targets\SonarQube.Integration.targets")
      IFileWrapper.Exists("c:\project\bin\targets\SonarQube.Integration.targets")
      IFileWrapper.Copy("C:\workspace\sonar-scanner-msbuild\tests\SonarScanner.MSBuild.PreProcessor.Test\bin\Debug\net48\Targets\SonarQube.Integration.targets", "c:\project\bin\targets\SonarQube.Integration.targets", True)
```

Reply:
I don't know if it's due to setting up the tests or a different behavior in net48 vs net5

Reply:
it's because I copied blindly without realizing the targets file name was different